prowler/providers/azure/services/containerregistry/containerregistry_admin_user_disabled/containerregistry_admin_user_disabled.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "Provider": "azure",
+  "CheckID": "containerregistry_admin_user_disabled",
+  "CheckTitle": "Disable Admin User for Azure Container Registry",
+  "CheckType": [],
+  "ServiceName": "containerregistry",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "",
+  "Severity": "high",
+  "ResourceType": "ContainerRegistry",
+  "Description": "WARNING: Enabling the admin user for Azure Container Registry (ACR) can pose security risks as it grants unrestricted access to the registry. It is recommended to disable the admin user and utilize Role-Based Access Control (RBAC) instead.",
+  "Risk": "If the admin user is enabled, it may lead to unauthorized access to the container registry and its resources, which could compromise the confidentiality, integrity, and availability of the images stored within.",
+  "RelatedUrl": "https://docs.microsoft.com/en-us/azure/container-registry/administrative-user",
+  "Remediation": {
+    "Code": {
+      "CLI": "az acr update --name <RegistryName> --resource-group <ResourceGroupName> --admin-user-enabled false",
+      "NativeIaC": "",
+      "Other": "",
+      "Terraform": "resource \"azurerm_container_registry\" \"example\" { admin_enabled = false }"
```

Comment:
```suggestion
      "Terraform": ""
```

Reply:
Why delete the example?

Reply:
For now we are not hardcoding Terraform in this field, it is commonly used to add a link with the Terraform code. I think it could be weird and in a future could be changed but now I think it is better to leave it empty.